docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -0,0 +1,166 @@
+# WebView
+
+The WebView functionality provides various features and primarily allows you to access web pages and other web content in your application.
+
+The [Tizen.WebView](https://samsung.github.io/TizenFX/latest/api/Tizen.WebView.html) namespace contains various features to enable web browsing, such as loading, displaying web pages, and navigating through the browsing history.
+
+To create a simple web browser, use the following web features:
+
+1. [Initializing and Finalizing WebView](#initialize)
+
+2. [Creating WebView Object](#create)
+
+3. [Loading Web Page](#load)
+
+4. [Handling Load Events](#event)
+
+5. [Setting Cookie Options](#cookie)
+
+## Prerequisites
+
+Ensure the following to enable WebView in your application:
```

Comment:
@younghajung 
Request you to kindly remove this line.

Reply:
Done